### DISCUSSION TITLE
Примеры запросов DoH для Postman

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#3590] [Q&A]
Всем привет,

есть ли примеры запросов для получения DNS-ответа по протоколу DoH в Postman? Я не нашел примеры запросов и коды ответов в документации к OpenAPI. Это вообще возможно если требуется Media Type `application/dns-message`?🤨

Спасибо.

**Comment by ameshkov** [comment]
Не уверен что он это поддерживает, там base64 строку надо передать в теле запроса

**Reply by ammnt** [reply]
@ameshkov, в URI? Ну URL в Postman то менять можно, а запрос в base64 заэнкодить при помощи того же самого base64decode.org. Формат тела запроса надо знать всё равно в raw/txt или json...

**Reply by ameshkov** [reply]
Можно и в URI, можно в теле POST-запроса

```
echo -n 'q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB' | base64 -D | curl -H 'content-type: application/dns-message' --data-binary @- https://cloudflare-dns.com/dns-query -o - | hexdump
```
